.github/workflows/test.yaml

```diff
@@ -34,7 +34,7 @@ permissions:
 jobs:
   test:
     name: ${{ matrix.type }} ${{ matrix.flavor }}
-    runs-on: ubuntu-latest
+    runs-on: uds-swf-ubuntu-big-boy-8-core
```

Comment:
@zachariahmiller  I did test uds-swf-ubuntu-big-boy-4-core and was seeing different tasks fail consistently. After moving to the 8 core I still see failure sometimes. I'll implement the logic you suggest here and see how it goes, thanks!

Reply:
Hmm - that sounds like it might be a different issue if it was still failing on the 8 core - this package is a dep package for many of our other packages so if resource reqs are doubling we would really want to find out why and reduce them back down to not have cascading consequences.

Reply:
@Racer159 Apologies, I didn't mean to indicate I see failures on the 8 core. I'll update my comment. With all the tests on the 8 core runner I had no intermittent failures.

Reply:
I'm seeing good success rates on the 4 core runner at this point with the tests. The Tag and release is already set to use a 4 core runner on publish, so I think we are good there.